### PR TITLE
soe-437: Creating and updating Readme, changelog, and license files.

### PR DESCRIPTION
**Pull request by cjwest**
@boznik , let me know if this makes sense. Since this feature hasn't had much testing or use, and we're getting errors in the logs, I'd like to keep this at an alpha or beta. 

Actually, I'd like to see if we can have it disabled on installation.... but that's for another conversation.

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Stanford Related Content x.x-x.x, xxxx-xx-xx
 ------------------------------------
 
-Stanford Related Content 7.x-1.0-alpha1, 2015-05-06
+Stanford Related Content 7.x-1.0-alpha, 2015-05-06
```
Comment (sherakama) [inline CHANGELOG.txt]: no need to be alpha. Release a point version.

File: modules/stanford_related_events/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,274 @@
+GNU GENERAL PUBLIC LICENSE
```
Comment (sherakama) [inline modules/stanford_related_events/LICENSE.txt]: Only need a license file at the top of every module. Sub modules are exempt.

**Comment by sherakama** [comment]
Looks good. Just the comments above about setting all of the .info files to version 7.x-1.0 and to remove the extra license files.

**Comment by cjwest** [comment]
Thanks @sherakama!

**Comment by boznik** [comment]
@cjwest plz resolve conflicts.

**Comment by sherakama** [comment]
Ah, sorry. That was me. I created a release for this for the JSE product. @cjwest, my guess is that the versions in your files have all changed. Please update to what was set in the main branch and re-release the tag.

**Comment by cjwest** [comment]
@boznik & @sherakama 

Oops! I didn't realize that this hadn't been merged before we started tagging. What do I put as version numbers 7.x-1.0?

**Comment by sherakama** [comment]
@cjwest,

Just merge in the latest from the main branch in to this branch and use the main branch's version.

**Comment by boznik** [comment]
@cjwest this branch still has unresolved conflicts

**Comment by cjwest** [comment]
@boznik : dealing with conflicts is still and art that I have yet to master.
same song, third verse...

**Pull request merged**